features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,200 @@
+---
+description: >-
+  A comprehensive guide that covers how you can add images and text over
```

Comment:
we can add a mention of 'new layer syntax' here and maybe how it supports nesting + some more features. People checking this page out would expect to see a different description from the old overlays page

Reply:
Changed to
>   A comprehensive guide that covers how you can add images and text over
  a base image, using the new layer syntax that supports overlay nesting
  and provides better positional control.

Reply:
looks good